### ISSUE TITLE
[BUG] cugraph is importing libgdf_cffi, resulting in an import error

### ISSUE THREAD

**Issue by rlratzel** [rapidsai/cugraph#275]
**Describe the bug**
Various cugraph modules are importing libgdf_cffi, which no longer exists. *This needs to be resolved for 0.7*

**To Reproduce**
`py.test cugraph/python`
```
ImportError while importing test module '/rapids/cugraph/python/cugraph/triangle_count/test_triangle_count.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
cugraph/triangle_count/test_triangle_count.py:18: in <module>
    import cugraph
cugraph/jaccard/jaccard_wrapper.pyx:19: in init cugraph
    from libgdf_cffi import libgdf
E   ModuleNotFoundError: No module named 'libgdf_cffi'
```

**Expected behavior**
All tests pass

**Comment by jakirkham** [comment]
Appears to be solved with PR ( https://github.com/rapidsai/cugraph/pull/276 ).

**Comment by BradReesWork** [comment]
closed by PR #278
